azure-monitor-opentelemetry-distro/samples/tracing/simple.py

```diff
@@ -4,10 +4,14 @@
 # license information.
 # --------------------------------------------------------------------------
 
-from azure.monitor.opentelemetry.distro import configure_opentelemetry
+from azure.monitor.opentelemetry.distro import configure_azure_monitor
 from opentelemetry import trace
 
-configure_opentelemetry()
+configure_azure_monitor(
+    connection_string="<your-connection-string>",
+    service_name="foo_service",
+    tracing_export_interval=15,
```

Comment:
I believe this is in milliseconds. Did you mean 15000?

Reply:
We may also want to consider including the scale in the name, but if not, documentation will cover it.